### PR TITLE
Story3

### PR DESCRIPTION
**Pull request by 2naCan**

### CODE REVIEW

**Review by JBHarrison5 — APPROVED** [review]

File: loginForm.php [hunk 1]
```diff
@@ -10,22 +10,26 @@
...
+    <div class='formDiv'>
+        <form class="form" action="login.php" method="post">
+  <span class="input-span">
+  <label for="username" class="label">Username</label>
+  <input type="text" name="username" id="username" required></span>
+            <span class="input-span">
+  <label for="password" class="label">Password</label>
+  <input type="password" name="password" id="password" required></span>
+            <input class="submit" type="submit" value="Log in">
+            <span class="span">Don't have an account? <a href="registrationForm.php">Sign up</a></span>
```
Comment (matt-ioa) [inline loginForm.php]: Indentation is a bit dodgy here

File: navbar.php [hunk 2]
```diff
@@ -11,24 +11,25 @@
...
-    <img id="logo" src="#">
+    <img id="logo" src="img/gamelog-high-resolution-logo-transparent.png">
 
     <div class="navButtons">
         <button>
             <a id="home" class="" href="index.php">Home</a>
         </button>
         <button>
-            <a id="userLibrary" class="" href="library.php">Your Library</a>
+            <a id="userLibrary" class="" href="library.php">Library</a>
```
Comment (matt-ioa) [inline navbar.php]: Why the empty classes here?

File: registrationForm.php [hunk 3]
```diff
@@ -7,17 +7,21 @@
...
+  <label for="username" class="label">Username</label>
+  <input type="text" name="username" id="username" required></span>
+            <span class="input-span">
+  <label for="steamID" class="label">Steam ID</label>
+  <input type="text" name="steamID" id="steamID" required></span>
+            <span class="input-span">
+  <label for="password" class="label">Password</label>
+  <input type="password" name="password" id="password" required></span>
+            <input class="submit" type="submit" value="Log in">
+            <span class="span">Already have an account?<a href="loginForm.php">Log in</a></span>
```
Comment (matt-ioa) [inline registrationForm.php]: Also badly indented

File: review.php [hunk 4]
```diff
@@ -3,23 +3,33 @@
...
 $userGames = $db->prepare('SELECT * FROM `userData` INNER JOIN `gameDisplay` ON `userData`.`appID` = `gameDisplay`.`appID` WHERE `userData`.`steamID` = \'' . $_SESSION['user'] . '\' ');
 $userGames->execute();
 $userGames = $userGames->fetchAll();
 
+
 foreach ($userGames as $game) {
-    echo "<div class='review'>";
-            echo "<h2>" . $game['gameName'] . "</h2>";
-            echo "<img src='" . $game['image'] . "/>";
+    if ($game['deleted'] === null) {
```
Comment (matt-ioa) [inline review.php]: Why not just add this to the WHERE clause of your SQL instead?

**Review by matt-ioa — CHANGES_REQUESTED** [review]
Main change is moving the deleted filter to the SQL rather than PHP

**Review by matt-ioa — APPROVED** [review]
:shipit:

**Pull request merged**